src/components/TransformWorkflow/FlowEditor/Components/LowerSectionTabs/StatisticsPane.tsx

```diff
@@ -366,11 +366,15 @@ export const StatisticsPane: React.FC<StatisticsPaneProps> = ({ height }) => {
   };
 
   const fetchRowCountAndColumns = async (schema: string, table: string) => {
-    const count = await httpGet(session, `warehouse/table_count/${schema}/${table}`);
-    setRowCount(count.total_rows);
+    try {
+      const count = await httpGet(session, `warehouse/table_count/${schema}/${table}`);
```

Comment:
but why are we asking for rows and columns of a table which doesn't exist

the error is in the backend, a try-catch here won't stop the error from happening